app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -988,7 +989,15 @@ public boolean isUserBlockedFromCommons() {
                 if (json == null) {
                     return null;
                 }
-                return gson.fromJson(json, FeedbackResponse.class);
+                Timber.d(json);
+                try {
+                    return gson.fromJson(String.valueOf(response.body()), FeedbackResponse.class);
+                }
+                catch (Exception e){
+                    return new FeedbackResponse("",0,0,0,new FeaturedImages(0,0),0,"",0);
+                }
+
```

Comment:
Can you please explain why did you add this part?

Reply:
What about returning null and handling this null variable where it will be used?